src/utils/slack/base.js

```diff
@@ -287,6 +289,58 @@ const getHlxConfigMessagePart = (hlxConfig) => {
   return `, _HLX Version_: *${hlxVersion}*, _Dev URL_: \`https://${rso.ref}--${rso.site}--${rso.owner}.aem.live\``;
 };
 
+/**
+ * Parses flags from a Slack message.
+ * Supports:
+ *   - `--flag` (boolean)
+ *   - `--key=value` (key-value pairs)
+ *   - `--key "multi word value"` (spaces in values)
+ *   - Handles dashes in flag names (e.g., `--my-flag`)
+ *
+ * @param {string} text - The message text from Slack.
+ * @returns {Object} - Parsed flags as key-value pairs.
+ */
+const parseFlags = (text) => {
```

Comment:
i am not sure we need such complex arg parsing? as i see it there's three possible things:
- onboard called with a single base URL
- onboard called with a file
- onboard optionally with profile x

check other commands for possibly simpler / dumb args usage.

Reply:
I went for flag parsing because I felt it could be more "readable" by the user. The flag/arg of profile is anyways optional in all the possibilities.
My idea was:
```
@spacecat onboard http://example.com 12345@AdobeOrg
```
and it would apply the `default` profile, or
```
@spacecat onboard http://example.com 12345@AdobeOrg --profile=summit
```
and apply the `summit` profile.

As a simpler alternative, we could still pass it as argument like the URL and the IMS Org ID

Reply:
@solaris007 understood and makes sense, sounds good to me ;)